packages/curriculum-compiler-json/index.sandbox.js

```diff
@@ -4,72 +4,127 @@ const ast = {
   type: 'root',
```

Comment:
(This file is for local debugging. Ignore it during the review process... This file doesn't affect anything else)